configuration/amd64/docker-compose.yml

```diff
@@ -46,8 +46,8 @@ services:
       - ./logs:/var/log/tmx
       - ./MAP:/var/www/plugins/MAP
   port_drayage_webservice:
-    image: usdotfhwaops/port-drayage-webservice:latest
-    container_name: port_drayage_webservice
+    image: usdotfhwaops/port-drayage-webservice:develop
+    container_name: port_drayage_webservicedock
```

Comment:
Was this "port_drayage_webservicedock" intentional?

Reply:
No great catch